docs/source/api/python/data.rst

```diff
@@ -57,3 +57,14 @@ Protein-Protein Interaction dataset
 
 .. autoclass:: PPIDataset
     :members: __getitem__, __len__
+
+Molecular Graphs
+````````````````
+
+To work on molecular graphs, make sure you have installed `RDKit <https://www.rdkit.org/docs/Install.html>`__.
+
+.. autofunction:: dgl.data.molecule.default_atom_featurizer
```

Comment:
`dgl.data.molecule.atom_featurizer(f=default_one)`?

Reply:
I'm now directly passing an `atom_featurizer` to `mol2dgl`. Think this like a user defined message function. I think we probably don't need a two-level wrapping here.